test/gtest/ucp/test_ucp_mmap.cc

```diff
@@ -375,8 +375,10 @@ UCS_TEST_P(test_ucp_mmap, alloc_mem_type) {
             ASSERT_UCS_OK(status);
 
             is_dummy           = (size == 0);
-            expect_rma_offload = ((*mem_type != UCS_MEMORY_TYPE_CUDA_MANAGED) &&
-                                  (is_tl_rdma() || is_tl_shm()));
+            expect_rma_offload = !UCP_MEM_IS_CUDA_MANAGED(*mem_type) &&
+                                 (is_tl_rdma() || is_tl_shm()) &&
+                                 !(UCP_MEM_IS_CUDA(*mem_type) &&
+                                  !check_reg_mem_types(sender(), *mem_type));
```

Comment:
why need to change this test? the "expect_rma_offload" should not be used for new protocols